packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -103,6 +103,9 @@
 # decode_DTS=0x00000000
 # decode_DDP=0x00000000
 
+# enable rc-core driver for RPi-style GPIO IR receiver at GPIO pin 18
+dtoverlay=ir-gpio
```

Comment:
This will only be included in new systems, anyone who has any existing install would have to add this manually.